platform/ui/src/components/InputNumber/InputNumber.tsx

```diff
@@ -14,6 +15,7 @@ import './InputNumber.css';
 
 const sizesClasses = {
   sm: 'w-[45px] h-[28px]',
+  bg: 'w-[206px] h-[35px]',
```

Comment:
if by bg you mean big? I prefer lg to be more consistent with the rest of the code